clang-tools-extra/test/clang-doc/test-path-relative.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: cp "%s" "%t/test.cpp"
+// RUN: clang-doc --doxygen --executor=standalone -p %t %t/test.cpp -output=../docs
+// RUN: FileCheck %s -input-file=%t/docs/index_json.js
+// RUN: rm -rf %t
+
+// CHECK: var RootPath = "{{.*}}../docs";
```

Comment:
You don't need this file. All the checks can go in the other file. You can separate them w/ different identifiers w/ `--check-prefixes=FOO` in `FileCheck` commands. If you look at almost any test folder in LLVM you'll see examples. Also, https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-FileCheck-check-prefix.

Also, don't copy files that aren't needed. `%s` is good enough for all the cases that I see.